packages/scrum-client/src/content/Profile/Profile.js

```diff
@@ -20,9 +33,42 @@ export default function Profile() {
     generateReferralLink(userDetails);
   }, [userDetails]);
 
+  const deleteProfile = async event => {
+    event.preventDefault();
+    try {
+      Request.post(
+        `${API_CONSTANTS.API_BASE_URL}${END_POINTS.API}${END_POINTS.DELETE_PROFILE}`,
+        userDetails
+      ).then(response => {
+        if (response) {
+          logout(dispatch);
+        } else {
+          // TODO what to do?
```

Comment:
To show a message to the user , an error has occurred while deleting your profile . This could be done later on

Reply:
ok